images/sdn/Dockerfile.rhel

```diff
@@ -1,10 +1,11 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
 WORKDIR /go/src/github.com/openshift/sdn
 COPY . .
-RUN make build --warn-undefined-variables
+RUN CGO_ENABLED=0 make build --warn-undefined-variables
```

Comment:
I think this is going to break the FIPS people, since openshift-sdn won't be using the hacked ssl.

Reply:
Aha. We still have this problem in current openshift-sdn. I'll make sure the fips people know.